common/src/main/java/org/opencds/cqf/common/helpers/ClientHelper.java

```diff
@@ -0,0 +1,51 @@
+package org.opencds.cqf.common.helpers;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.rest.client.api.IGenericClient;
+import ca.uhn.fhir.rest.client.api.ServerValidationModeEnum;
+import ca.uhn.fhir.rest.client.interceptor.BasicAuthInterceptor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+@Component
+public class ClientHelper {
+
+    @PostConstruct
+    public void init() {
+        ClientHelper.environment = this.environmentTemp;
+    }
+
+    @Autowired
+    public Environment environmentTemp;
+
+    public static Environment environment;
+
+    /* TODO - depending on future needs:
+            1.  add OAuth
+            2.  change if to switch to accommodate additional FHIR versions
+     */
+    private static IGenericClient getRestClient(String whichFHIRVersion, String url){
```

Comment:
Let's have this take a FhirContext rater than a fhir version. The context is already version specific.